sale_stock_available_to_promise_release/reports/sale_order.xml

```diff
@@ -9,15 +9,15 @@
     >
       <xpath expr="//th[@name='th_description']" position="after">
         <th
-                t-if="doc.state != 'draft'"
+                t-if="doc.state != 'draft' and doc.env['ir.config_parameter'].sudo().get_param('sale_stock_available_to_promise_release.display_is_available')"
```

Comment:
```suggestion
                t-if="doc.state != 'draft' and doc.company_id.display_is_available"
```
As mentioned above if you configure by company then you can do something like that.